### PR TITLE
fix: improve layout of persona list

### PR DESCRIPTION
**Pull request by UncleBill**
## Compare

A row can be fully filled

| before | after |
| -- | -- |
|  image | image |

Shadow of the card shows now

| before | after |
| -- | -- |
| image | image |

### CODE REVIEW

File: packages/maskbook/src/extension/options-page/DashboardRouters/Personas.tsx [hunk 1]
```diff
@@ -14,24 +14,27 @@ import { extendsTheme } from '../../../utils/theme'
...
+            gridGap: theme.spacing(3),
             [theme.breakpoints.down('sm')]: {
-                margin: 0,
-                paddingLeft: 0,
+                display: 'block',
             },
         },
+        personaItem: {
+            listStyle: 'none',
+            marginBottom: theme.spacing(2),
```
Comment (UncleBill) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Personas.tsx:36]: Should I increase it to `theme.spacing(3)`?

File: packages/maskbook/src/_locales/ja/messages.json [hunk 2]
```diff
@@ -292,6 +292,7 @@
     "plugin_wallet_captcha_hint": "上の式の答えを入力してください",
     "plugin_wallet_select_a_token": "トークンを選択",
     "plugin_wallet_connect_a_wallet": "ウォレットと接続",
+    "plugin_wallet_unlock_metamask": "",
```
Comment (UncleBill) [inline packages/maskbook/src/_locales/ja/messages.json:295]: Looks like someone skip the commit hook before.

File: packages/maskbook/src/extension/options-page/DashboardRouters/Personas.tsx [hunk 3]
```diff
@@ -96,15 +101,19 @@ export default function DashboardPersonasRouter() {
...
-                        ))}
+                    <ul className={classes.personaList}>
+                        {personas
+                            .sort((a, b) => {
+                                if (a.updatedAt > b.updatedAt) return -1
+                                if (a.updatedAt < b.updatedAt) return 1
+                                return 0
+                            })
+                            .map((persona) => (
+                                <li key={persona.identifier.toText()} className={classes.personaItem}>
```
Comment (septs) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Personas.tsx:112]:
```suggestion
                            .map((persona, key) => (
                                <li key={key} className={classes.personaItem}>
```

Comment (UncleBill) [reply packages/maskbook/src/extension/options-page/DashboardRouters/Personas.tsx:112]: Index as a key is an anti-pattern.

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**